tests/python/PyImportSortBearTest.py

```diff
@@ -21,3 +29,81 @@
      'import abc\nimport xyz\n'),
     settings={'known_standard_library_imports': 'xyz',
               'known_first_party_imports': 'abc'})
+
+
+class PyImportSortBearTest(unittest.TestCase):
+
+    def setUp(self):
+        self.queue = Queue()
+        self.section = Section('PyImportSortBear')
+        self.uut = PyImportSortBear(self.section,
+                                    self.queue)
+
+    def test_treat_seperated_imports_independently(self):
+        self.file = (
```

Comment:
no need for `self.` here, as the attribute isnt used outside of the method.

Also normally these sample files are declared as module level variables, as they can often be reused in multiple tests.